lib/tasks/posts.rake

```diff
@@ -134,3 +134,16 @@ task 'posts:remap', [:find, :replace] => [:environment] do |_,args|
   end
   puts "", "#{i} posts remapped!", ""
 end
+
+def find_each_with_recent(given_scope, recent_date: 6.months.ago)
```

Comment:
Note sure, maybe add a parameter? That way people can tell the rake task what "recent" mean?